clang/docs/ReleaseNotes.rst

```diff
@@ -165,6 +165,8 @@ C++23 Feature Support
 - Added a separate warning to warn the use of attributes on lambdas as a C++23 extension
   in previous language versions: ``-Wc++23-lambda-attributes``.
 
+- Implemented `P2448R2: Relaxing some constexpr restrictions <https://wg21.link/P2448R2>`_.
+
```

Comment:
Can you say that it is implemented as a conforming extension?
We probably should document it in `LanguageExtensions.rst`

The feature macro in clang was already bumped, so that doesn't need change

Reply:
Okay, done. I'm not sure I updated `LanguageExtensions.rst`, please take a look.
I also added some testing for enabling extension in earlier language modes.